ncl/ncl_entries/climatology_functions.ipynb

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
<p>The width of the plot is cutting off the title values</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Good catch.  

I'll go in and add some custom titles that won't take up so much space.     

I could probably add the semicolon after the plot as well to avoid the matplotlib lines above the figures as well. 

Removing the figures altogether is another option (and would be more concise), but I think I'll leave that call for later.